docs/reference/enable-gpu.rst

```diff
@@ -1,46 +1,153 @@
-Enable Nvidia GPUs
+.. _nvidia-gpu:
+
+Enable NVIDIA GPUs
```

Comment:
This whole page is not a Reference - it is a how-to guide. We need to move it there. In general, Reference should be kept for factual information. When related to software, we usually mean describing APIs, functions, options for commands, this kind of stuff.

Reply:
ACK! Moving it under how-to guides